bin/webpack.js

```diff
@@ -168,7 +170,23 @@ yargs.parse(process.argv.slice(2), (err, argv, output) => {
 		argv["display"] = "verbose";
 	}
 
-	var options = require("./convert-argv")(yargs, argv);
+	try {
+		Error.stackTraceLimit = 30;
+		var options = require("./convert-argv")(yargs, argv);
+	} catch(err) {
```

Comment:
This catches all errors. "Normal" errors from configuration should not be affected by this code. Check error.